test/build/base-url-homepage/base-url-homepage.test.js

```diff
@@ -1,18 +1,26 @@
-const fs = require('fs');
 const path = require('path');
 const cheerio = require('cheerio');
+const {setupBuildTest, readFiles} = require('../../test-utils');
 
-const html = fs.readFileSync(path.join(__dirname, 'build', 'index.html'), 'utf8');
```

Comment:
But with that setup script comes a caveat: **none of these files exist when the test loads, so you‘ll have to read files in `beforeAll(() => …`

Reply:
I actually prefer this, touching globally shared test values is just asking for one test to modify them and then create unexpected behavior.

As we move to use the programatic interface & more "one fixture per test" tests, a separate `files` object would most likely be created for each test, so this is a good step in that direction.